package.json

```diff
@@ -14,14 +14,15 @@
   "scripts": {
     "build": "node ./build.js",
     "generate": "node --experimental-modules ./generator/index.mjs",
-    "all": "npm run generate & npm run build",
+    "all": "npm run generate; npm run build",
```

Comment:
```suggestion
    "all": "npm run generate && npm run build",
```
perhaps?

Reply:
Yes. Better. Only runs the second command if the first succeeds.
The original single ampersand was making me run the build twice.